com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -293,6 +298,8 @@ public override bool Equals(object obj)
         Dictionary<RegId, List<Chunk>>  m_Registry = new Dictionary<RegId, List<Chunk>>();
 
         internal Dictionary<int, Cell> cells = new Dictionary<int, Cell>();
+        private Dictionary<int, CellChunkInfo> chunkInfo = new Dictionary<int, CellChunkInfo>();
```

Comment:
Private member should be named "m_Stuff" and private keyword is not necessary here.

Reply:
yeah but was internal, I added private as a test to see if I could've made it private but then forgot to change the naming and remove the keyword. Will change this.